### PR TITLE
475: Add international standing order consents functional tests

### PR DESCRIPTION
**Pull request by frlaura-jianu**
Issue: https://github.com/securebankingaccesstoolkit/securebankingaccesstoolkit/issues/475
Description: Added international standing order consents functional tests.

### CODE REVIEW

**Comment by dbadham-fr** [comment]
Can you add the junit/v3_1_9 and junit/v3_1_10 test classes please?

For all new APIs we want to have tests in place for 3.1.8/9/10

File: src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/international/standing.orders/consents/legacy/LegacyGetInternationalStandingOrderConsentsTest.kt [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+        // Then
+        assertThat(result).isNotNull()
+        assertThat(result.data).isNotNull()
+        assertThat(result.risk).isNotNull()
+        assertThat(result.data).isEqualTo(consent.data)
+        assertThat(result.risk).isEqualTo(consent.risk)
+    }
+
+    /*@EnabledIfVersion(
```
Comment (jorgesanchezperez) [inline src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/international/standing.orders/consents/legacy/LegacyGetInternationalStandingOrderConsentsTest.kt:234]: Please delete the commented code.

**Review by jorgesanchezperez — CHANGES_REQUESTED** [review]
Please, Delete commented code.

File: src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/international/standing.orders/consents/legacy/LegacyGetInternationalStandingOrderConsentsTest.kt (hunk 1 above)
Comment (frlaura-jianu) [reply src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/international/standing.orders/consents/legacy/LegacyGetInternationalStandingOrderConsentsTest.kt:234]: Oops, missed that ! I will remove the comments.

**Comment by frlaura-jianu** [comment]
Sure, I will add the test classes for junit/v3_1_9 & v3_1_10. I always forget about them. Thanks !

**Comment by frlaura-jianu** [comment]
> Can you add the junit/v3_1_9 and junit/v3_1_10 test classes please?
> 
> For all new APIs we want to have tests in place for 3.1.8/9/10

Sure, I will add the test classes for junit/v3_1_9 & v3_1_10. I always forget about them. Thanks !

**Review by dbadham-fr — APPROVED** [review]

**Review by jorgesanchezperez — APPROVED** [review]

**Pull request merged**